packages/web3-eth-accounts/src/models/Account.js

```diff
@@ -102,7 +102,7 @@ export default class Account {
      * @returns {EncryptedKeystoreV3Json | {version, id, address, crypto}}
      */
     encrypt(password, options) {
-        return Account.fromPrivateKey(this.privateKey, this.accounts.transactionSinger).toV3Keystore(password, options);
+        return Account.fromPrivateKey(this.privateKey, this.accounts.transactionSigner).toV3Keystore(password, options);
```

Comment:
This method still fails for me when I fix the typo, because `this.accounts` is `undefined` when created via `web3.eth.accounts.create()` or `web3.eth.accounts.wallet.create(1)`. I solved by defaulting `this.accounts` to `{}` in the constructor.